### PR TITLE
Czech translation

### PR DESCRIPTION
**Pull request by Zdendaki**
I've created czech translation.

### CODE REVIEW

File: VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+<language name="Czech" localizedName="Čeština">
+	<translation id="project">Projekt</translation>
+	<translation id="file">Soubor</translation>
+	<translation id="solution">Řešení</translation>
+
+	<translation id="noActiveProject">Žádný aktivní projekt</translation>
+	<translation id="noActiveFile">Žádný aktivní soubor</translation>
+	<translation id="noActiveSolution">Žádné aktivní řešení</translation>
+
+	<translation id="NoneAssetSlot">None</translation>
```
Comment (Ryavell) [inline VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml:12]: The "NoneAssetSlot" key does not seem to have been translated.
Should there be a "Zádné" here?

**Review by Ryavell — CHANGES_REQUESTED** [review]
Thanks for the new translation. 
But there seems to be no translation for NoneAssetSlot

File: VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml (hunk 1 above)
Comment (Zdendaki) [reply VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml]: Yes, my bad, I'm sorry

File: VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<language name="Czech" localizedName="Čeština">
+	<translation id="project">Projekt</translation>
+	<translation id="file">Soubor</translation>
+	<translation id="solution">Řešení</translation>
+
+	<translation id="noActiveProject">Žádný aktivní projekt</translation>
+	<translation id="noActiveFile">Žádný aktivní soubor</translation>
+	<translation id="noActiveSolution">Žádné aktivní řešení</translation>
+
+	<translation id="NoneAssetSlot">Žádné</translation>
```
Comment (Zdendaki) [inline VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml:12]: Fixed

**Review by Zdendaki — COMMENTED** [review]
Fixed

File: VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml (hunk 2 above)
Comment (Ryavell) [reply VisualStudioDiscordRPC.Shared/Translations/cs-CZ.xml:12]: Good

**Review by Ryavell — APPROVED** [review]

**Pull request merged**